scripts/mbedtls_dev/bignum_mod_raw.py

```diff
@@ -109,7 +109,18 @@ def result(self) -> List[str]:
         result = (self.int_a * self.r_inv) % self.int_n
         return [self.format_result(result)]
 
+class BignumModRawModNegate(bignum_common.ModOperationCommon,
+                            BignumModRawTarget):
+    """ Test cases for mpi_mod_raw_neg(). """
+    test_function = "mpi_mod_raw_neg"
+    test_name = "Modular negation: "
+    symbol = "(-A)"
```

Comment:
Shouldn't the symbol just be `-`?

Reply:
I was trying to not replicate the symbol of subtraction. This is for debugging purposes only anyhow.

Reply:
Well, it's what goes into the test case. Since `-` is the symbol for negation, and the test name includes "modular", I don't think there's any risk of confusion